op-e2e/actions/interop/proofs_test.go

```diff
@@ -707,14 +709,33 @@ func runChallengerTest(gt *testing.T, test *transitionTest, actors *dsl.InteropA
 	}
 }
 
-func WithInteropEnabled(actors *dsl.InteropActors, agreedPrestate []byte, disputedClaim common.Hash, claimTimestamp uint64) fpHelpers.FixtureInputParam {
+func WithInteropEnabled(t helpers.StatefulTesting, actors *dsl.InteropActors, agreedPrestate []byte, disputedClaim common.Hash, claimTimestamp uint64) fpHelpers.FixtureInputParam {
 	return func(f *fpHelpers.FixtureInputs) {
 		f.InteropEnabled = true
 		f.AgreedPrestate = agreedPrestate
 		f.L2OutputRoot = crypto.Keccak256Hash(agreedPrestate)
 		f.L2Claim = disputedClaim
 		f.L2BlockNumber = claimTimestamp
 
+		chainIndexA, err := actors.ChainA.ChainID.ToUInt32()
+		require.NoError(t, err)
+		chainIndexB, err := actors.ChainB.ChainID.ToUInt32()
+		require.NoError(t, err)
+		deps := map[eth.ChainID]*depset.StaticConfigDependency{
+			actors.ChainA.ChainID: {
+				ChainIndex:     supervisortypes.ChainIndex(chainIndexA),
```

Comment:
I expected the chain index to be the offset into the dependency for the chain rather than the actual chain ID. Although it seems very common to do it this way in tests - I'm not really sure why.

Reply:
We don't really know what the chain index will be, but that makes sense.
For tests, it suffices for the chain index to be unique, which is why the chainID is used instead.